packages/common/src/EncryptionHelper.js

```diff
@@ -18,6 +18,7 @@ function computeVoteHash(request) {
     { t: "int", v: request.salt },
     { t: "address", v: request.account },
     { t: "uint", v: request.time },
+    { t: "bytes", v: "0x" },
```

Comment:
This change makes all downstream usages of `EncryptionHelper` work be default; with nothing other than this blank ancillary data.

Reply:
For now, we will simply not support ancillary data in the voter dapp.

Reply:
SGTM